### DISCUSSION TITLE
Feature: no-future-dates for QDate

### DISCUSSION THREAD

**Discussion by Hashen110** [quasarframework/quasar#12378] [Ideas / Proposals] [answered]
Is it possible to add no-future-dates (or another prop name) for QDate to disable users selecting future dates? or add navigation-max-year-month-day prop?

**Comment by MilosPaunovic** [comment, marked as answer]
I don't really see it happening as you can achieve this behaviour with the use of existing `options` prop:
[https://codepen.io/milospaunovic/pen/OJObYLo?editors=1011](https://codepen.io/milospaunovic/pen/OJObYLo?editors=1011
)

[https://quasar.dev/vue-components/date#limiting-options](https://quasar.dev/vue-components/date#limiting-options)

**Reply by Hashen110** [reply]
Problem solved, Thanks :)
